analyse_SiW.py

```diff
@@ -0,0 +1,132 @@
+from ROOT import TH1F, TTree, TChain, TFile, TF1, TCanvas, TGraphErrors, TPaveText
```

Comment:
same as above. Do you by any chance mean analogue when you name files SiW? I find it quite misleading, as digital is SiW as well, isn't it? (or Pb)